extensions/apps/extensions/src/components/pages/info-page/info-page.tsx

```diff
@@ -63,151 +82,285 @@ export const InfoPage: React.FC<InfoPageProps> = ({ appId }) => {
+                          alternativeAvatars={appData.author?.akashaProfile?.avatar?.alternatives?.map(
+                            alternative => transformSource(alternative),
+                          )}
+                        />
+                        <Icon
+                          icon={<ChevronRightIcon />}
+                          size="sm"
+                          color={{ light: 'secondaryLight', dark: 'secondaryDark' }}
+                          customStyle="ml-auto"
+                        />
+                      </Stack>
+                    </Card>
+                  )}
+                </Section>
+                {appData.gallery?.length && (
+                  <Section
+                    dividerPosition={DividerPosition.Top}
+                    title={t('Gallery')}
+                    viewMoreLabel={t('View All')}
+                    onClickviewMoreLabel={() => {
+                      setShowImageGalleryOverlay(!showImageGalleryOverlay);
+                    }}
+                  >
+                    <ExtensionImageGallery
+                      images={appData.gallery?.map(gImage => ({
+                        ...gImage,
+                        src: transformSource(gImage.src).src,
+                      }))}
+                      showOverlay={showImageGalleryOverlay}
+                      toggleOverlay={() => setShowImageGalleryOverlay(!showImageGalleryOverlay)}
+                    />
+                  </Section>
+                )}
+
+                <Section title={t('General Information')} dividerPosition={DividerPosition.Top}>
+                  <Stack spacing="gap-y-2">
+                    <Stack direction="row" justify="between">
+                      <Text variant="body2" color={{ light: 'grey4', dark: 'grey7' }}>
+                        {t('Package name')}
+                      </Text>
+                      <Button variant="text" size="md" label={appData.name} />
+                    </Stack>
+                    <Divider />
+                    <Stack direction="row" justify="between">
+                      <Text variant="body2" color={{ light: 'grey4', dark: 'grey7' }}>
+                        {t('Extension ID')}
+                      </Text>
+                      <Button variant="text" size="md" label={truncateDid(appData.id)} />
+                    </Stack>
+                    <Divider />
+                    <Stack direction="row" justify="between">
+                      <Text variant="body2" color={{ light: 'grey4', dark: 'grey7' }}>
+                        {t('Latest update')}
+                      </Text>
+                      <Button
+                        variant="text"
+                        size="md"
+                        label={formatRelativeTime(latestRelease?.node?.createdAt)}
+                        onClick={() => {
+                          navigate({
+                            to: '/info/$appId/audit-log',
```

Comment:
is this path handled? what should it display?

Reply:
all the navigation to the subroutes is broken... you can check this on next. I need to debug to see what is going on

Reply:
also the subroutes will not be included in the current task, it's just too much to add there but I'll try to at least fix the routing